streams-kafka/src/main/java/io/kipe/streams/kafka/processors/StatsBuilder.java

```diff
@@ -72,6 +72,13 @@
  * value Serde, and a topic base name "topic-base-name". The statistics expression is then added using the "with"
  * method, with the target field specified as "myCount". Finally, the topology is built with a grouping based on the
  * "group" field, and the output is a KeyValueStore with a key of type "String".
+ * <p>
+ * The table below shows the stats command with its stateful and internal topics details:
+ * <pre>
+ * | command | stateful | internal topics                  |
+ * |---------|----------|----------------------------------|
+ * | stats   | yes      | {topicsBaseName}-processor-store |
```

Comment:
We should rename the topic to `{topicsBaseName}-processor-store`. Please do it as part of this PR.